Gems/EditorPythonBindings/Code/Source/ActionManager/PythonActionManagerHandler.cpp

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Source/ActionManager/PythonActionManagerHandler.h>
```

Comment:
update: This is less bad as we're in the .cpp, but just wanted to double check should `PythonActionManagerHandler` be in `Source`? Or would it be better in an include folder under `ActionManager`?